src/openrct2/Date.h

```diff
@@ -44,3 +69,28 @@ namespace OpenRCT2
         static int32_t GetDaysInMonth(int32_t month);
     };
 } // namespace OpenRCT2
+
+struct RealWorldDate
+{
+    uint8_t day;
+    uint8_t month;
+    int16_t year;
+    uint8_t day_of_week;
+};
+
+struct RealWorldTime
+{
+    uint8_t second;
+    uint8_t minute;
+    uint8_t hour;
+};
+
+extern const int16_t days_in_month[MONTH_COUNT];
+extern OpenRCT2::Date gDate;
```

Comment:
We should reduce the amount of globals, its better to get this variable over the context, I know its a bit ugly at the moment to have to use GetContext() but that will help refactor passing the context eventually down to each corresponding Update function.

Reply:
You can add a helper function for the time being something like in the Date object:
```cpp
        static Date& GetDate()
        {
            auto* gameState = GetContext()->GetGameState();
            return gameState->GetDate();
        }
```
Then all uses can be replaced with `Date::GetDate()` until we have passing of the Context/GameState refactored